### PR TITLE
PR for ( Deployment, Development, Automation ) Microservice.

### LINKED ISSUE #65: Deployment Dockerfiles ( Dockerfile ) — backend-microservices-authentication_service

### LINKED ISSUE #66: Deployment Dockerfiles ( Dockerfile ) — backend-microservices-player_service

### LINKED ISSUE #67: Deployment docker-compose.yaml

### LINKED ISSUE #68: Development Dockerfiles ( Dockerfile.dev ) — backend-microservices-authentication_service

### LINKED ISSUE #69: Development Dockerfiles ( Dockerfile.dev ) — backend-microservices-player_service

### LINKED ISSUE #70: Development Dockerfiles ( Dockerfile.dev ) — frontend

### LINKED ISSUE #71: Development docker-compose.dev.yaml

### LINKED ISSUE #72: Deployment Dockerfiles ( Dockerfile ) — frontend

### LINKED ISSUE #74: Makefile automate the Deployment, Development

### PR DESCRIPTION
**Pull request by ychaaibi**
This pull request will deploy the microservices and frontend as container to docker, Deployment, Development, Automation

### **Devops :**

**Deploy microservices as containers to Docker :**

- **_Deployment_**

  - resolve #65
  - resolve #66
  - resolve #72
  - resolve #67

* **_Development_**

  - resolve #68
  - resolve #69
  - resolve #70
  - resolve #71

- **_Automation_**

  -  resolve #74

### CODE REVIEW

File: docker-compose.dev.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
+services:
+    frontend:
+        build:
+            context: ./frontend/
+            dockerfile: Dockerfile.dev
+        image: frontend:develop
+        container_name: frontend-develop
+        ports:
+            - 81:81
```
Comment (zwzone) [inline docker-compose.dev.yaml:9]: `80:81` is better than `81:81`

File: docker-compose.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
+services:
+    frontend:
+        build:
+            context: ./frontend/
+            dockerfile: Dockerfile
+        image: frontend:deploy
+        container_name: frontend-deploy
+        ports:
```
Comment (zwzone) [inline docker-compose.yaml:8]: The same thing in here

**Review by zwzone — APPROVED** [review]

File: docker-compose.yaml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+            - 81:81
```
Comment (reogine) [inline docker-compose.yaml:9]: the same thing that zwzone mentioned here

**Review by reogine — APPROVED** [review]

File: docker-compose.yaml (hunk 3 above)
Comment (ychaaibi) [reply docker-compose.yaml:9]: Will be done...

File: docker-compose.dev.yaml (hunk 1 above)
Comment (ychaaibi) [reply docker-compose.dev.yaml:9]: Will be done...

**Pull request merged**